apps/composer-ui/tailwind.config.js

```diff
@@ -0,0 +1,15 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    './app/**/*.{js,ts,jsx,tsx,mdx}',
```

Comment:
Is it possible to use glob patterns to simplify these

Reply:
i haven't done yet, but we can consider later on